shared/ui-components/src/atoms/__snapshots__/ApplicationStatus.stories.storyshot

```diff
@@ -21,11 +17,30 @@ exports[`Storyshots Listing Sidebar/Application Status Due Soon And Vivid 1`] =
         },
         "applicationDueDate": "2030-04-11T16:00:00+00:00",
         "applicationFee": "30.0",
+        "applicationMethods": Array [
+          Object {
+            "acceptsPostmarkedApplications": true,
+            "type": "POBox",
+          },
+          Object {
+            "type": "LeasingAgent",
+          },
+          Object {
+            "type": "PaperPickup",
+          },
+        ],
         "applicationOpenDate": "",
         "applicationOrganization": "98 Archer Street",
         "applicationPhone": "(408) 217-8562",
+        "assets": Array [
+          Object {
+            "fileId": "https://regional-dahlia-staging.s3-us-west-1.amazonaws.com/listings/archer/archer-studios.jpg",
+            "label": "building",
+            "referenceId": "Uvbk5qurpB2WI9V6WnNdH",
+            "referenceType": "Listing",
+          },
+        ],
         "attachments": Array [],
```

Comment:
Since attachments are no longer in use I guess those arrays can be removed, correct?

Reply:
Yes, please.